components/ConnectPolkadotWalletButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { polkadotWalletModalOpenAtom } from "@/store/polkadot";
+import { Button } from "./ui/button";
+import { useAtom } from "jotai";
+
+export function ConnectPolkadotWalletButton() {
+  const [, setPolkadotWalletModalOpen] = useAtom(polkadotWalletModalOpenAtom);
```

Comment:
This could be a custom hook, but does what you want it to do.

Reply:
There is a way to combine these two components. The SelectWallet component from talismn and the button component. 
https://github.com/TalismanSociety/talisman-connect/tree/master/packages/connect-components#walletselect
The trigger component. Will probably look into doing something like that in future. Or maybe deprecate this completely in favor of wallet connect for polkadot.